docs/USAGE.md

```diff
@@ -0,0 +1,80 @@
+# Usage
+
+- [How do I interact with the app?](#interacting-with-the-app)
+- [How do I stop and start the server?](#stopping-and-starting-the-server)
+- [How do I run tests?](#running-tests)
+- [How do I rebuild Ember?](#rebuilding-ember)
+- [Do I need special environment variables?](#environment)
+- [How do I generate and serve documentation?](#generating-documentation)
+- [How do I push changes to GitHub?](#pushing-changes)
+
+## Interacting with the app
+
+You'll generally interact with the app using `ember server`. If you are running the [Elixir API](https://github.com/code-corps/code-corps-api/) locally, running `ember server` will make requests to the API by default. 
+If you aren't running the API locally, you can use our remote development enviornment by running `ember server --environment=remote-development`.
+
+Once the server has been started, you can visit your app at `http://localhost:4200`.
+
+Note that Ember is able to live-reload as changes are made to the codebase. Ember-CLI will build those changes and apply them; No need to stop and start the server for every change!
+
+## Stopping and starting the server
+
+Need to stop the server? `Ctrl+C` will do the trick.
+
+To start the server again run `ember server`.
+
+## Rebuilding Ember
+
+To rebuild, simply run `ember server`. You can clear npm's cache with `npm --cache-clear`, then reinstall your node modules with `npm install` if necessary.
+
+## Running tests
+
+We use [ember-exam](https://github.com/trentmwillis/ember-exam) for running tests since it allows for parallel testing, randomized orders, and other neat configurations. You can check out the repository readme there to see the available options. A few handy ones to use when running tests locally are outlined below.
+
+* `ember exam` will run the tests
+* `ember exam --split=3 --weighted --parallel` will run tests in 3 PhantomJS instances in parallel with an equal split. Currently, 3 parallel instances will run the tests the quickest.
+* `ember exam --random` will run the tests in a random order
+* `ember exam --filter='acceptance'` will only run acceptance tests. The same sytax can be used for other types of tests, such as `ember exam --filter='unit'` and `ember exam --filter='integration'`
+
+## Environment
```

Comment:
This requires some rewrite but is good to mention in conjunction with API docs. We won't have much need for a developer to have environment variables _in Ember_, but they should be aware of how the environment can come into contact with the API.